Utilities/General/python/cmssw_das_client.py

```diff
@@ -66,12 +67,15 @@ def get_value(data, filters, base=10):
       else:
         yield values
 
-def get_data(query, limit=None, threshold=None, idx=None, host=None):
+def get_data(query, limit=None, threshold=None, idx=None, host=None, cmd="das_client"):
```

Comment:
hi @vkuznet - it doesn't seem that the cmd should be overridden if it has been provided by the calling code. Perhaps the default of cmd should be None and then if cmd==None the code below figures out if dasgoclient is available and otherwise falls back to das_client.